src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,31 @@
      * Андрей - 600
      * София - 900</p>
      */
+    static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+        StringBuilder myResult = new StringBuilder();
```

Comment:
let's rename to `result` or `resultInfo` or smth else
use 'my..." isn't good practice